Doc/whatsnew/3.7.rst

```diff
@@ -1,3 +1,5 @@
+.. _whatsnew37:
```

Comment:
This _seems_ like a thing I should be able to do in Sphinx without defining the label myself, but I couldn't figure it out. Happy to change it if there's some other way to link to the page that I've missed.